piton/design/chip/tile/rtl/tile.v.pyv

```diff
@@ -827,289 +827,57 @@ if (TILE_TYPE == `BLACKPARROT_TILE) begin : g_blackparrot_core
-        .ready_o                            (transducer_ready_lo),
-        // OpenPiton Bus
-        // BP -> L1.5
-        .miss_v_i                           (arb_miss_v_lo),
-        .miss_yumi_o                        (arb_miss_yumi_li),
-        .uncached_i                         (arb_uncached_lo),
-        .miss_addr_i                        (arb_miss_addr_lo),
-        .lru_way_i                          (arb_miss_lru_lo),
-        .store_i                            (arb_is_store_lo),
-        .store_data_i                       (arb_miss_data_lo),
-        .size_op_i                          (arb_miss_size_lo),
-
-        // L1.5 -> BP
-        .l15_transducer_ack                 (l15_transducer_ack),
-        .l15_transducer_header_ack          (l15_transducer_header_ack),
-
-        .transducer_l15_rqtype              (transducer_l15_rqtype),
-        .transducer_l15_amo_op              (transducer_l15_amo_op),
-        .transducer_l15_size                (transducer_l15_size),
-        .transducer_l15_val                 (transducer_l15_val),
-        .transducer_l15_address             (transducer_l15_address),
-        .transducer_l15_data                (transducer_l15_data),
-
-        .transducer_l15_nc                  (transducer_l15_nc),
-        .transducer_l15_threadid            (transducer_l15_threadid),
-        .transducer_l15_prefetch            (transducer_l15_prefetch),
-        .transducer_l15_blockstore          (transducer_l15_blockstore),
-        .transducer_l15_blockinitstore      (transducer_l15_blockinitstore),
-        .transducer_l15_l1rplway            (transducer_l15_l1rplway),
-        .transducer_l15_invalidate_cacheline(transducer_l15_invalidate_cacheline),
-        .transducer_l15_csm_data            (transducer_l15_csm_data),
-        .transducer_l15_data_next_entry     (transducer_l15_data_next_entry),
-
-        .l15_transducer_val                 (l15_transducer_val),
-        .l15_transducer_returntype          (l15_transducer_returntype),
-
-        .l15_transducer_data_0              (l15_transducer_data_0),
-        .l15_transducer_data_1              (l15_transducer_data_1),
-
-        .transducer_l15_req_ack             (transducer_l15_req_ack),
-
-        .data_mem_pkt_o                     (arb_data_mem_pkt_li),
-        .data_mem_pkt_v_o                   (arb_data_mem_pkt_v_li),
-        .data_mem_pkt_yumi_i                (arb_data_mem_pkt_yumi_lo),
-
-        .tag_mem_pkt_o                      (arb_tag_mem_pkt_li),
-        .tag_mem_pkt_v_o                    (arb_tag_mem_pkt_v_li),
-        .tag_mem_pkt_yumi_i                 (arb_tag_mem_pkt_yumi_lo),
-
-        .stat_mem_pkt_o                     (arb_stat_mem_pkt_li),
-        .stat_mem_pkt_v_o                   (arb_stat_mem_pkt_v_li),
-        .stat_mem_pkt_yumi_i                (arb_stat_mem_pkt_yumi_lo)
-    );
+    assign spc_grst_l = reset_rrrr;
+
+    // Unused signals - Tieing them to zero
```

Comment:
tie these to zero inside the bp_piton_top

Reply:
that way we can change / implement them without needing to modify OP

Reply:
Yup, on it.